app/models/host/base.rb

```diff
@@ -199,7 +199,15 @@ def set_interfaces(parser)
       # is saved to primary interface so we match it in updating code below
       if !self.managed? && self.primary_interface.mac.blank? && self.primary_interface.identifier.blank?
         identifier, values = parser.suggested_primary_interface(self)
-        self.primary_interface.mac = Net::Validations.normalize_mac(values[:macaddress]) if values.present?
+        if values.present?
+          self.primary_interface.mac = Net::Validations.normalize_mac(values[:macaddress])
+          # bridge interfaces are not attached to parent interface so save would not be possible
+          if interface_class(identifier) != Nic::Bridge
+            self.primary_interface.virtual = values[:virtual]
```

Comment:
this can be nil, you probably need to `!!values[:virtual]`

Reply:
Amended this change blindly and crossing fingers.